.github/workflows/script.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 header_test = 'Released under a MIT (SEI)-style license'
-header = 'Copyright 2021 Carnegie Mellon University. All Rights Reserved. \n Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.'
+header = 'Copyright 2022 Carnegie Mellon University. All Rights Reserved. \n Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.'
```

Comment:
This week, I added the following code in the Treasury apps to always grab the current year.  I was planning to share this with everyone at the next weekly retro.

import os
import datetime

header_test = 'Released under a MIT (SEI)-style license'
header = 'Copyright ' + str(datetime.date.today().year) + ' Carnegie Mellon University. All Rights Reserved. \n Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.'

Reply:
Cool, thanks!